documentation/analytics/recipes/daily-active-users.md

```diff
@@ -1,2 +1,162 @@
+    ConsumerGroupName = "YOUR_CONSUMER_GROUP_GOES_HERE",
+    StorageConnectionString = "YOUR_STORAGE_ACCOUNT_CONNECTION_STRING_GOES_HERE",
+    LeaseContainerName = "NAME_OF_LEASE_CONTAINER_GOES_HERE"
+};
+
+var listener = new EventHubsListener(listenerConfig);
+```
+
+### 2.2 Setup Message Parser
+
+(TODO: Make sure the below code is accurate and up to date)
+```cs
+// Setup Message Parser
+var parser = new EventHubListenerMessageJsonParser(new ConsoleCorruptMessageHandler());
+```
+
+### 2.3 Setup Output Managers
+```cs
+// Setup Output Managers
+var outputManager = new DataLakeStoreOutputManager(domain, webApp_clientId, clientSecret, subscriptionId, adlsAccountName);
+```
+
+### 2.4 Setup Message Router
+```cs
+// Build up the Router Pipeline
+var builder = new MessageRouterBuilder();
+var filePathAlgorithm = new PipelineDateFilePathAlgorithm(newFileOption: NewFileNameOptions.Every5Minutes);
+
+// Setting up "Daily Active Users Recipe"
+var dauSerializer = new CsvOutputFormatter("id", "type", "version", "gameSession", "enqueueTimeUtc", "gamerTag");
+
+builder.Pipeline("dau")
+    .HandlesMessageType("session-start", "1.0.0")
+    .OutputTo(new ConsoleOutputManager(dauSerializer)
+        , new FileOutputManager(dauSerializer, filePathAlgorithm, @"C:\dev\USQLDataRoot")
+        , new DataLakeStoreOutputManager(
+            dauSerializer,
+            filePathAlgorithm,
+            serviceClientCretentials,
+            _configuration[NAH_Azure_SubscriptionId],
+            _configuration[NAH_Azure_DLSOutputManager_AccountName])
+        );
+        builder.DefaultPipeline
+            .AddHandler(new RandomIntMessageHandler())
+            .OutputTo(new ConsoleOutputManager(new CsvOutputFormatter()));
+
+var router = builder.Build();
+```
+
+### 2.5 Setup Message Processor
+```cs
+var messageProcessor = new MessageProcessor<EventHubJsonMessage>(listener, parser, router);
+
+
+// Run in an async context since main method is not allowed to be marked as async
+Task.Run(async () =>
+{
+    await messageProcessor.ProcessAndBlockAsync();
+}).GetAwaiter().GetResult();
+```
+
+### 3 Setup Job Queries
+The purpose of the DAU.usql query is to calculate the daily active users of your game over time. A daily active user is a distinct count of any user, using your game within a single UTC time, day period (no matter how many times a single user visits the game that day). The single query DAU.usql described and linked below could be run:
+* Once a __day__ to compute a new DAU value (1 Date, 1 Count)
+* Or on a __weekly__ basis (7 Dates, 7 counts)
+* Or on a __monthly__ basis (28-31 Dates, 28-31 Counts)
+
+
+| Query that should be run           | By what service                           |
+|------------------------------------|-------------------------------------------|
+| [DAU.usql](../../../src/Nether.Analytics.DataLake.Jobs/DAU.usql) | Azure Data Lake Analytics |
+
+#### 3.1 Setup and Schedule Query1
+
+* Open the Nether.Analytics Solution
```

Comment:
Scip this section for now. We are just about to have that code ready, so even though I do like the ilustrations and the way you described executing the job, we are not going to need it. It's all going to be code :-)